status_api/README.md

```diff
@@ -0,0 +1,48 @@
+# NVSS API Status App (API and UI Build)
+
+## Dependencies
+ - .NET Core 6.100
+ - Docker with MSSQL Server
+
+## Quick Start
+
+1. The status_api uses models, migrations, and database from `messaging`. Follow its [README](../README.md) to get started
+and run a few API calls to populate its database.
+
+2. Go to the `status_ui/` folder and build the frontend:
+
+```shell
+npm install
+npm run build
+```
```

Comment:
I don't think this is quite right, because the status_ui should only need to be built if there are changes since the results of the build actually get checked into source control. I think this README should 1) mention that the status_ui is prebuilt in files that are located in the status_api/StatusUI directory and 2) point to the status_ui/README.md as instructions for updating the UI frontend if needed.